xapian-core/tests/api_anydb.cc

```diff
@@ -575,6 +575,11 @@ DEFINE_TESTCASE(topercent1, backend) {
     return true;
 }
 
+#if 0
+/** This testcase uses glass backend, even when backend intened to test is
+ *  inmemory or other. This testcase may fail when run in parallel, hence
+ *  disabling for now.
+ */
 // tests the percentage values returned
 DEFINE_TESTCASE(topercent2, backend) {
     BackendManagerLocal local_manager(test_driver::get_srcdir() + "/testdata/");
```

Comment:
`git grep BackendManagerLocal` shows two other testcases which ought to have similar treatment.

Reply:
Done. I had this in mind, but somehow other two test cases never failed, so decided to just keep them.